dpnp/tests/test_mathematical.py

```diff
@@ -1197,6 +1197,23 @@ def test_left_right_args(self, dtype):
         result = dpnp.interp(ix, ixp, ifp, left=-40, right=40)
         assert_dtype_allclose(result, expected)
 
+        # left and right as ndarray
```

Comment:
It can be done through `parametrize("left, right", [[-40, 40], [dpnp.array(-40), dpnp.array(40)]])` and then:
```python
expected = numpy.interp(x, xp, fp, left=get_array(numpy, left), right=get_array(numpy, right))
result = dpnp.interp(ix, ixp, ifp, left=left, right=right)
```
or something similar